src/components/Sidebar/sidebarLists.js

```diff
@@ -60,6 +60,12 @@ export const sideBarList = [
     route: "/json-converter",
     tags: "Conversion,Syntax,JSON",
   },
+  {
+    name: "ASCII-Hex Converter",
+    iconClass: "bi bi-hexagon",
+    route: "/ascii-to-hex",
+    tags: "Conversion,Syntax,JSON",
```

Comment:
Please add the tags "ASCII" and "HEXADECIMAL" here and remove syntax and json from the list. The rest looks fine.

Reply:
@vaibhav-kaushal test and merge fast,  I am maintaining and updating this branch everyday with the main daily for 2 weeks. Can't do it for really long.